lib/btrfs_test.pm

```diff
@@ -4,27 +4,29 @@ use base 'consoletest';
 use strict;
 use testapi;
 
-=head2 unpartitioned_disk_in_bash
+=head2 set_playground_disk
 
-Choose the disk without a partition table for btrfs experiments.
-Defines the variable C<$disk> in a bash session, which defaults to
-'/dev/*b' drive (i.e. /dev/{vd,xvd,sd}b) be it blank or used before.
+Returns disk without a partition table for filesystem experiments.
+Sets the test variable C<PLAYGROUNDDISK>, on first invocation of
+the function.
 =cut
-sub set_playground_disk_in_bash {
-    my $vd = 'vd';    # KVM
-    if (check_var('VIRSH_VMM_FAMILY', 'xen')) {
-        $vd = 'xvd';
+sub set_playground_disk {
+    unless (get_var('PLAYGROUNDDISK')) {
```

Comment:
Generally it is good style to prefer too high level of indention. Can you just turn this around?

E.g.

```perl
my $disk = get_var('PLAYGROUNDDISK');
return $disk if $disk;
…
```